src/components/Tooltip/Tooltop.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { Tooltip } from './Tooltip'
+
+export default {
+  title: 'Components/Tooltip',
+  component: Tooltip,
+  parameters: {
+    info: `
+    USWDS 2.0 Tooltip component
+
+    Source: https://designsystem.digital.gov/components/tooltip/
+    `,
+  },
+}
+
+export const tooltipDefault = (): React.ReactElement => (
+  <div className="margin-4">
+    <Tooltip label="Default">Default</Tooltip>
+  </div>
+)
+
+export const tooltipTop = (): React.ReactElement => (
+  <div className="margin-4">
+    <Tooltip position="top" label="Top">
+      Show on top
+    </Tooltip>
+  </div>
+)
+
+export const tooltipBottom = (): React.ReactElement => (
+  <div className="margin-4">
+    <Tooltip position="bottom" label="Bottom">
+      Show on bottom
+    </Tooltip>
+  </div>
+)
+
+export const tooltipRight = (): React.ReactElement => (
+  <div className="margin-4">
+    <Tooltip position="right" label="Right">
+      Show on right
+    </Tooltip>
+  </div>
+)
+
+export const tooltipLeft = (): React.ReactElement => (
+  <div className="margin-4">
+    <Tooltip position="left" label="Left">
+      Show on left
+    </Tooltip>
+  </div>
+)
+
+export const CustomComponent = (): React.ReactElement => {
+  type MockTooltipProps = React.PropsWithChildren<{
+    to: string
+    className?: string
+  }> &
+    JSX.IntrinsicElements['a'] &
+    React.RefAttributes<HTMLAnchorElement>
+  const CustomLink: React.ForwardRefExoticComponent<MockTooltipProps> = React.forwardRef(
```

Comment:
@suzubara curious on your thoughts here. We had both not used forwardRef before. This seems like the suggested way to get a ref to a child component's html element. I had seen some discussion about forwarding refs on DatePicker and FileInput so I wanted to call this one out for discussion specifically.

For people who just want to wrap an existing component, this _does_ seem cumbersome as they would need to expose a ref in order to use this component.

Would you recommend going this route or a different appraoch?